lxd/images.go

```diff
@@ -3455,6 +3455,8 @@ func imageExportPost(d *Daemon, r *http.Request) response.Response {
 			return fmt.Errorf(opWaitAPI.Err)
 		}
 
+		d.State().Events.SendLifecycle(projectName, lifecycle.ImagePushed.Event(fingerprint, projectName, op.Requestor(), log.Ctx{"target": req.Target}))
```

Comment:
I think this should use `ImageRetrieved` as it's really the same from an audit point of view (the full image data was transferred out).